djredcap/management/subcommands/inspect.py

```diff
@@ -50,6 +51,11 @@ class Command(BaseCommand):
     requires_model_validation = False
     db_module = 'django.db'
     args = 'filename'
+    # Add a command line option -m allowing the user to set a filename for models.
+    # Filename is relative to the current working directory.
+    option_list = BaseCommand.option_list + \
+        (make_option('-m', '--model-file', dest='model_filename',
```

Comment:
Change formatting to match [above](https://github.com/chop-dbhi/django-redcap/pull/24/files#diff-444fbd30bc80e17a0ad228900070e20eR44).